database/__init__.py

```diff
@@ -20,3 +21,4 @@ async def async_init(self):
 
 def setup(bot):
     bot.db = DatabaseExtension(bot)
+    bot.add_cog(MongoCog(bot))
```

Comment:
This seems to be a different way of adding cogs than what we've previously done, was that intentional?

Reply:
I'm taking the `Cog` subclass and calling `bot.add_cog(MyCog(bot))`, as before. The `setup` function isn't in the same file as the cog, since the mongo cog is part of the database extension. I guess I could add a `setup` to `mongo` and add `load_extension(database.mongo)`? but this way made most sense to me

Reply:
Very sorry, that's just me completely misremembering what we did elsewhere.

Reply:
That's entirely my mistake, I forgot how we did this in the rest of the code.